### PR TITLE
Review overhauled Chair Documentation

### PR DESCRIPTION
**Pull request by kimdhamilton**
Overhaul to DIF chair documentation:

chair_readme.md:
- Created as a new index file to guide chairs through available resources

chair_cheat_sheet.md:
- Restructured for readability to make a quick reference before, during, and after meetings
- Move out and link to duplicated info

chair_guidelines.md:
- Summarize for readability
- Move out and link to duplicated info

chair_reference.md:
- Consolidated all detailed references and resources
- Added sections on training resources, technical resources, and working group processes

agenda_files.md:
- Created as a comprehensive guide for managing agenda.md files
- Included details on purpose, structure, and best practices
- Added section on HackMD integration with setup tips and screenshots

All documents:
- Ensured consistent cross-linking between documents
- Improved overall structure and readability
- Eliminated duplications and redundancies
- Standardized formatting and style across all files

### CODE REVIEW

File: agenda_files.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+   - Action items or followups
+   - Substantive discussion outcomes
+
+**Note**: Keep `agenda.md` notes brief, as meetings are recorded with transcripts. Focus on major points only.
+
+**Tip**: Use GitHub "issues" for detailed work and progress tracking. Reference these issues in meeting notes rather than duplicating information.
+
+## Structure and Content
+
+DIF uses GitHub repo templates which include a starting point agenda.md file you can use. See [Credential Schemas agenda.md](https://github.com/decentralized-identity/credential-schemas/blob/main/AGENDA.md) for an example of a populated file.
```
Comment (andorsk) [inline agenda_files.md:19]: Love it.

**Review by andorsk — APPROVED** [review]
This looks fantastic. Thank you for the amazing work @kimdhamilton . This is really really helpful.

**Comment by andorsk** [comment]
Approved from my. Please let @ankurdotb also review before we merge.

**Review by peacekeeper — APPROVED** [review]
Looks good.

Personally I found the HackMD integration too complicated with sometimes unexpected behaviors and conflicts, so now I prefer to just edit the agenda.md directly on Github. But I guess that's up to each work item / working group to decide.

**Comment by kimdhamilton** [comment]
I'm going to merge. If others have feedback post-merge, please let me know

**Pull request merged**